tunnelsdk/api.go

```diff
@@ -20,7 +21,8 @@ type ClientRegisterRequest struct {
 }
 
 type ClientRegisterResponse struct {
-	Version TunnelVersion `json:"version"`
+	Version   TunnelVersion `json:"version"`
+	PollEvery time.Duration `json:"poll_every"`
```

Comment:
How about `ReregisterAt time.Time` instead, and then make the client loop sleep until that time after each request

Reply:
Since the client will be running on machines we don't control and possibly with out of date clocks, I think it's probably safer to use a duration here